src/screens/discover/DiscoverDetailScreen.tsx

```diff
@@ -71,34 +61,76 @@ const requestHandler = async (props: RequestHandlerProps) => {
 };
 
 const DiscoverDetailScreen: React.FC = () => {
+    const query: string = useLoaderData() as string;
     const { profile, setProfile } = useProfileContext();
     const path = window.location.pathname
         .match(/\/\w+$/)
         ?.join()
         .slice(1);
-    const [data, setData] = useState<ShowData[] | null>(null);
     const [loading, setLoading] = useState<boolean>(true);
+    const storageItem = localStorage.getItem('streamabilityView');
+    const initialView = storageItem === 'grid' ? 'grid' : 'list';
+    const [viewState, setViewState] = useState<'list' | 'grid'>(initialView);
+    const [hash, setHash] = useState<number>(1);
+    const {
+        data,
+        setData,
+        loading: dataLoading,
+        moreToFetch,
+        refetch,
+    } = usePaginatedData({ query: query });
```

Comment:
<img width="1435" alt="image" src="https://github.com/user-attachments/assets/08c12625-eeb1-49e7-9ff8-71d5b8b11c3e">
The data seems to be broken in the PR preview. I think it would simplify this PR to remove the pagination, and instead just keep the data state we had before. The pagination can be added in #838

Reply:
remove the pagination for the discover detail screen.